src/Site/views/languageforge/theme/default/sass/languageforge_default.scss

```diff
@@ -2,13 +2,10 @@
 @import '_variables.scss';
 
 // Pull in Bootstrap files - by default this loads ALL features of Bootstrap which we may not need
-@import '../../../../../../../../vendor_bower/bootstrap/scss/bootstrap.scss';
+@import '../.././../../../../vendor_bower/bootstrap/scss/bootstrap.scss';
```

Comment:
You're right, it's a typo, though it's not a missing `.`, but rather a `./` should be removed (it works how it is, but it's redundant).

About removing the import, are you thinking of Font Awesome?